go/test/endtoend/tabletgateway/vtgate_test.go

```diff
@@ -271,7 +271,6 @@ func TestReplicaTransactions(t *testing.T) {
 	require.NoError(t, err)
 	serving := replicaTablet.VttabletProcess.WaitForStatus("SERVING", 60*time.Second)
 	assert.Equal(t, serving, true, "Tablet did not become ready within a reasonable time")
-	utils.AssertContainsError(t, readConn, fetchAllCustomers, "not found")
```

Comment:
If the assertion has changed, let’s update it accordingly.

Reply:
Fixed via 89060c80363

Reply:
Modified more by 83dc5aa46a4